packages/ra-core/src/sideEffect/auth.ts

```diff
@@ -29,6 +29,12 @@ const nextPathnameSelector = state => {
 };
 
 const currentPathnameSelector = state => state.router.location;
+const errorMessageSelector = (error, defaultMessage) =>
```

Comment:
As this is a simple function, can you rename it to something like `getErrorMessage` ?

Reply:
Yes, will update.